target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -109,13 +109,14 @@ TARGET_DEVICES += ubnt_rocket-m
 define Device/ubnt_nanostation-m
   $(Device/ubnt-xm)
   DEVICE_TITLE := Ubiquiti Nanostation M
-  SUPPORTED_DEVICES += nanostation-m
+  SUPPORTED_DEVICES += nano-m,nanostation-m
```

Comment:
It's really not necessary to add `nanostation-m` and `nanostation-m-xw` to the list, as it's being added automagicaly from the `define Device/ubnt_nanostation-m`

Reply:
In other words, having just `nano-m` and `nano-m-xw` in `SUPPORTED_DEVICES` list should be enough

Reply:
BTW from my POV this commit should be probably squashed to your previous one (just one single commit) or if you want them to be separated (in two separate commits), you need to make changes to your both commits. This second commit is simply based on your first commit (there is no `SUPPORTED_DEVICES += nanostation-m` line in OpenWrt project source tree at this moment). I hope, that I've expressed myself clearly :-)

Reply:
When shorting the SUPPORTED_DEVICES, it really makes all look more nice, when all is put into a single commit.
What do you think now?